src/AuthService.Grpc/Infrastructure/Mappings/AuthServiceProfile.cs

```diff
@@ -22,7 +22,7 @@ public AuthServiceProfile()
 
             CreateMap<BusinessModels.Token, Token>()
                 .ReverseMap();
-            CreateMap<BusinessModels.User, User>()
+            CreateMap<AuthService.BusinessLogic.Entities.User, User>()
```

Comment:
AuthService.BusinessLogic.Entities not compiling. Please recheck the namespace.
![image](https://user-images.githubusercontent.com/46915707/223441621-541e308d-5e3e-400b-ba51-856d7787690c.png)